code/modules/power/cable.dm

```diff
@@ -402,6 +402,9 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list(new/datum/stack_recipe("cable restrain
 	var/obj/structure/cable/target_type = /obj/structure/cable
 	var/target_layer = CABLE_LAYER_2
 
+/obj/item/stack/cable_coil/full/Initialize(mapload)
+	. = ..(mapload, max_amount)
```

Comment:
What is this

Reply:
with the new way of saving starting items, we can't do cable_coil = new(src, 30) like before.
So this is how we can get a full stack of cables